app/controllers/embed_controller.rb

```diff
@@ -37,7 +37,7 @@ def topics
       end
     end
 
-    response.headers["X-Robots-Tag"] = "noindex, indexifembedded"
+    response.headers["X-Robots-Tag"] = "googlebot:noindex, indexifembedded"
```

Comment:
```suggestion
    response.headers["X-Robots-Tag"] = "googlebot:noindex,indexifembedded"
```

Small one as the example on the blog post doesn't seem to use a space between the values.